pubsub.go

```diff
@@ -271,6 +277,12 @@ func (c *PubSub) Ping(ctx context.Context, payload ...string) error {
 	return err
 }
 
+func (c *PubSub) SetHealthTimeout(t time.Duration) {
+	if t > 0 {
+		c.healthTimeout = t
+	}
+}
+
```

Comment:
`SetHealthTimeout` only makes sense for `Channel`, but is set on `PubSub`. It is confusing.